File: app/src/main/java/com/nolamarel/onlinelibrary/Adapters/myBooks/MyBook.java
package com.nolamarel.onlinelibrary.Adapters.myBooks;

public class MyBook {
    String bookImage, bookId, bookPercent;

    public MyBook(String bookImage, String bookId, String bookPercent) {
        this.bookImage = bookImage;
        this.bookId = bookId;
        this.bookPercent = bookPercent;
    }

    public String getBookImage() {
        return bookImage;
    }

    public void setBookImage(String bookImage) {
        this.bookImage = bookImage;
    }

    public String getBookId() {
        return bookId;
    }

    public void setBookId(String bookId) {
        this.bookId = bookId;
    }

    public String getBookPercent() {
        return bookPercent;
    }

    public void setBookPercent(String bookPercent) {
        this.bookPercent = bookPercent;
    }
}

File: app/src/main/java/com/nolamarel/onlinelibrary/Adapters/myBooks/MyBook.kt
package com.nolamarel.onlinelibrary.Adapters.myBooks

class MyBook {
    var bookId: Int
    var bookName: String? = null
    var bookAuthor: String? = null
    @kotlin.jvm.JvmField
    var bookImage: String
    var bookContent: String? = null

    constructor(bookImage: String, bookId: Int, bookName: String?, bookAuthor: String?) {
        this.bookImage = bookImage
        this.bookId = bookId
        this.bookName = bookName
        this.bookAuthor = bookAuthor
    }

    constructor(
        bookId: Int,
        bookName: String?,
        bookAuthor: String?,
        bookImage: String,
        bookContent: String?
    ) {
        this.bookId = bookId
        this.bookName = bookName
        this.bookAuthor = bookAuthor
        this.bookImage = bookImage
        this.bookContent = bookContent
    }

    constructor(bookId: Int, bookImage: String) {
        this.bookId = bookId
        this.bookImage = bookImage
    }
}
